### PR TITLE
[ADD] make report layout changes

### PR DESCRIPTION
**Pull request by siddharth7815**
Task Link : 

https://www.druidoo.io/web?db=druidoo-druidoo-master-349765&token=J39sYB1yfc1NMuVYgIBr#id=2090&action=367&model=project.task&view_type=form&menu_id=237

### CODE REVIEW

**Comment by siddharth7815** [comment]
@ivantodorovich .. can you please elaborate more on why travis CI build failed ?

So In future PR I will consider it.

File: account_check_deposit/report/report_checkdeposit.xml [hunk 1]
```diff
@@ -10,47 +10,57 @@
...
+                            <div class="col-4">
+                                <h3>Beneficiary:</h3>
+                                <span
+                                        class="col-sm-4"
+                                        t-field="o.company_id.partner_id"
+                                        t-options='{"widget": "contact", "fields": ["address", "name", "phone", "fax"], "no_marker": true}'
+                                />
+                            </div>
+                        </div>
+                        <div id="check_informations" class="row mt32 mb32">
```
Comment (ivantodorovich) [inline account_check_deposit/report/report_checkdeposit.xml]:
```suggestion
                        <div id="other_information" class="row mt32 mb32">
```

File: account_check_deposit/report/report_checkdeposit.xml [hunk 2]
```diff
@@ -10,47 +10,57 @@
...
+                            <span t-field="o.name"/>
+                        </h1>
+
+
+                        <div class="row">
+                            <div class="col-8">
+                                <h3>Bank:</h3>
+                                <span
+                                        t-field="o.bank_journal_id.bank_account_id.bank_id.name"
+                                />
```
Comment (ivantodorovich) [inline account_check_deposit/report/report_checkdeposit.xml]:
```suggestion
                                <span t-field="o.bank_journal_id.bank_account_id.bank_id.name"/>
```
Can we take the opportunity to write these spans like this?

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
In this case test are failing because you don't have pre-commit installed in your clone.
This is a requirement for OCA `13.0` branches

https://odoo-community.org/blog/the-oca-blog-1/post/black-isort-pre-commit-97

**Comment by ivantodorovich** [comment]
Where's the 12.0 PR?
You plan to do it after 13.0?

Also can you provide screenshots of the generated report?

**Pull request closed without merging**